docs/data_collection.rst

```diff
@@ -0,0 +1,99 @@
+Data Collection
+===============
+
+This document describes how snippets data is collected, stored and used.
+
+Snippets follows Mozilla's established best practices for data collection
+and storage of personally identifiable information.
+
+Data Collection: Impressions
+----------------------------
+
+Immpression data is sampled at a rate of ~10%. For a tracked impression the following
+information is sent to the stats server.
+
+.. code-block:: json
+
+    snippet_name
+    locale
+    country
+    metric
+    campaign
+
+
+This information is sent via ajax request to https://snippets-stats.mozilla.org/ . Metric for the above
+will always be `impression`.
+
+
+Data Collection: Clicks & Events
+--------------------------------
+
+Snippets may contain a link to a Mozilla destination. When they do the link may be constructed
+with UTM parameters that are used in Google Analytics. The site they land on (example: mozilla.org)
+reports these parameters to Google.
+
+This link when clicked sends the information to Google. An example links is:
+
+.. code-block:: json
+
+    https://www.mozilla.org/firefox/sync/?utm_source=desktop-snippet&utm_medium=snippet&utm_content=sync&utm_term=5274&utm_campaign=desktop&sample_rate=0.1&snippet_name=5274
+
+Snippets may also send specific events to our data warehouse. An example use is capturing that a
```

Comment:
Data warehouse is a pretty unclear term here. I feel like it's better to just refer to "Mozilla" as a black box that this data gets sent to, and describe our policies on that data.